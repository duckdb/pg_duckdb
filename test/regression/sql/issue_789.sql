SET duckdb.force_execution = TRUE;

COPY (SELECT 1 INTO frak UNION SELECT 2) TO '/blob';
COPY (SELECT 1 INTO frak UNION SELECT 2) TO '/blob.parquet';
