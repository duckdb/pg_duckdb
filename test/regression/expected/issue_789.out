SET duckdb.force_execution = TRUE;
COPY (SELECT 1 INTO frak UNION SELECT 2) TO '/blob';
ERROR:  COPY (SELECT INTO) is not supported
COPY (SELECT 1 INTO frak UNION SELECT 2) TO '/blob.parquet';
ERROR:  (PGDuckDB/DuckdbUtilityHook_Cpp) Executor Error: (PGDuckDB/MakeDuckdbCopyQuery) DuckDB COPY only supports SELECT statements
